### ISSUE TITLE
Output formatting

### ISSUE THREAD

**Issue by davishmcclurg** [davishmcclurg/json_schemer#27]
Implement the JSON Schema spec's "basic" output format: https://github.com/json-schema-org/json-schema-spec/pull/679

**Comment by jfeltesse-mdsol** [comment]
Very interested in this feature!

Just a heads up to let you know people are looking forward to using this.

**Comment by davishmcclurg** [comment]
@jfeltesse-mdsol good to know! Is the current output not working for you? I'm curious to hear how the spec's output format will be more useful.

**Comment by jfeltesse-mdsol** [comment]
It's about future proofing.
Right now I've not put your gem to use in any project, still exploring, but when I do so it'd be nice to know the code I'll write around won't be thrown away is 6 months.

Related to this we have other languages in use at my company and while making a cross-language shared wrapper library is probably not worth it, reviewing and maintaining other languages is going to be much easier if everyone processes the same kind of data structures, as standardized by the spec.

**Comment by obahareth** [comment]
@davishmcclurg This feature is also very interesting for me!

**Comment by davishmcclurg** [comment]
Released in 2.0.0. Use `output_format` option (classic/flag/basic/detailed/verbose):

```ruby
JSONSchemer.schema(schema).validate(data, :output_format => 'basic')
```
